### PR TITLE
JBPM-7118: Stunner - Mediators (zoom,pan) not working properly

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/JBPM-7118

Before:
before

After:
after

---

Part of an ensemble:
- https://github.com/kiegroup/lienzo-core/pull/19
- https://github.com/kiegroup/lienzo-tests/pull/15
- https://github.com/kiegroup/kie-wb-common/pull/1675

### CODE REVIEW

File: src/test/java/com/ait/lienzo/client/core/shape/wires/SelectionManagerTest.java [hunk 1]
```diff
@@ -61,4 +99,92 @@ public void testOnlyLeftMouseButtonCanStartSelection()
...
+        when(transform.getScaleX()).thenReturn(scaleX);
+        when(transform.getScaleY()).thenReturn(scaleY);
+        doReturn(x).when(manager).relativeStartX();
+        doReturn(y).when(manager).relativeStartY();
+        doReturn(manager).when(onMouseXEventHandler).getSelectionManager();
+        doNothing().when(manager).drawSelectionShape(anyInt(), anyInt(), anyInt(), anyInt(), any(Layer.class));
+
+        onMouseXEventHandler.drawSelectionShape(mouseEvent);
+
+        verify(manager).drawSelectionShape(eq(10d), eq(20d), eq(-30d), eq(-60d), eq(overLayer));
```
[the remark this answers is not in the document]
Comment (karreiro) [reply src/test/java/com/ait/lienzo/client/core/shape/wires/SelectionManagerTest.java]: Fixed, thanks.

File: src/test/java/com/ait/lienzo/client/core/shape/wires/SelectionManagerTest.java [hunk 2]
```diff
@@ -61,4 +99,92 @@ public void testOnlyLeftMouseButtonCanStartSelection()
...
+        when(transform.getScaleX()).thenReturn(scaleX);
+        when(transform.getScaleY()).thenReturn(scaleY);
+        doReturn(x).when(manager).relativeStartX();
+        doReturn(y).when(manager).relativeStartY();
+        doReturn(manager).when(onMouseXEventHandler).getSelectionManager();
+        doNothing().when(manager).drawSelectionShape(anyInt(), anyInt(), anyInt(), anyInt(), any(Layer.class));
+
+        onMouseXEventHandler.drawSelectionShape(mouseEvent);
+
+        verify(manager).drawSelectionShape(eq(10d), eq(20d), eq(1d), eq(1d), eq(overLayer));
```
[the remark this answers is not in the document]
Comment (karreiro) [reply src/test/java/com/ait/lienzo/client/core/shape/wires/SelectionManagerTest.java]: Fixed, thanks.

**Comment by karreiro** [comment]
@jomarko Thanks for the review. Changes applied.

**Comment by karreiro** [comment]
Thanks @hasys :-)

**Comment by romartin** [comment]
Merging as this one has been already merged -> https://github.com/kiegroup/kie-wb-common/pull/1675. It depends on this one.

**Pull request merged**